### DISCUSSION TITLE
google的這封郵件是什麼意思？是需要更新服務端嗎？

### DISCUSSION THREAD

**Discussion by caofengzi** [SeaHOH/GotoX#89] [一般]
[Action Required] Please upgrade your App Engine Standard applications to Go version 1.9 by November 1
   Hello Google App Engine Standard Environment Customer,

We’re writing to let you know that we are deprecating Go 1.6 and Go 1.8 support on App Engine, and new deployments using these language versions will no longer be available on November 1, 2018.

Go 1.6 and 1.8 are no longer supported and do not receive updates or security fixes from the open source Go Programming Language project. Per our deprecation policy, we will stop accepting new deployments that use these unsupported language versions on November 1, 2018.

What do I need to know?
Starting on November 1, 2018, you will no longer be able to deploy new versions of App Engine Standard applications that use Go 1.6 or Go 1.8. Applications that have already been deployed using Go 1.6 or Go 1.8 will continue to run for now. However, we recommend re-deploying with a supported language version in all cases.

We are in the process of scheduling the shut down of already deployed applications that use Go 1.6 and 1.8 in 2019. Later this year, we will be making a separate announcement.

What do I need to do?
Please re-deploy your App Engine Standard applications using Go 1.9 before November 1, 2018. You can learn more about upgrading your application at the Go Release History site, in particular about adopting the context package introduced to the standard library in Go 1.7.

The following projects linked with your account use a version of Go on App Engine that are affected by this deprecation:
如果是還請老大弄一個新服務端部署工具。

**Comment by SeaHOH** [comment]
暂时不需要

**Comment by caofengzi** [comment]
剛才用老大那個部署工具重新部署了一次，顯示go版本是1.8.7，希望以後弄一個go1.9版本的服務端部署程序。

**Comment by SeaHOH** [comment]
还有 3 个月的时间，而且不影响已部署 ID。

**Comment by yfdyh000** [comment]
https://github.com/Kisesy/gscan_quic/issues/147#issuecomment-410277527
